sql-jdbc/src/main/java/org/opensearch/jdbc/types/TimestampType.java

```diff
@@ -78,13 +79,17 @@ else if (value.charAt(23) == '+' || value.charAt(23) == '-') {
                 }
             }
 
-            if (calendar == null) {
-                return Timestamp.valueOf(value);
+            final Timestamp ts;
+            if (value.length() < 11) {
```

Comment:
question why 11 here? is it to check if value is in yyyy-MM-dd format?

Reply:
Yes. I can add a comment to clarify this.